tests/python/direct/test_repro.py

```diff
@@ -2169,7 +2214,8 @@ def fusion_func(fd: FusionDefinition) -> None:
             high=HIGH_VAL,
         ),
     ]
-    fd.validate(inputs)
```

Comment:
I thought `fd.validate(inputs)` automatically does validation, no?

Reply:
- `exec_nvfuser` tests the python repro and generates outputs.
- You can pass `reference_outputs` to `fd.validate` to avoid running the fusion twice.
- `fd.validate(inputs)` runs fusion then validates.
- `fd.validate(inputs, reference_outputs) just validates.